twingate/internal/provider/resource/resource.go

```diff
@@ -17,9 +17,10 @@ import (
 )
 
 var (
-	ErrPortsWithPolicyAllowAll      = errors.New(model.PolicyAllowAll + " policy does not allow specifying ports.")
-	ErrPortsWithPolicyDenyAll       = errors.New(model.PolicyDenyAll + " policy does not allow specifying ports.")
-	ErrPolicyRestrictedWithoutPorts = errors.New(model.PolicyRestricted + " policy requires specifying ports.")
+	ErrPortsWithPolicyAllowAll            = errors.New(model.PolicyAllowAll + " policy does not allow specifying ports.")
+	ErrPortsWithPolicyDenyAll             = errors.New(model.PolicyDenyAll + " policy does not allow specifying ports.")
+	ErrPolicyRestrictedWithoutPorts       = errors.New(model.PolicyRestricted + " policy requires specifying ports.")
+	ErrWildcardAddressWithEnabledShortcut = errors.New("not allowed to have both: wildcard address and enabled browser shortcut, please choose one of it")
```

Comment:
@ekampf @alexb-twingate here is error placeholder message

Reply:
Here's my take: "Resource with a CIDR range or wildcard can't have the browser shortcut enabled."

Reply:
thank you, update error message